flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -142,6 +180,60 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler {
     }
   }
 
+  private fun beginNativeFrames(result: Result) {
+    if (!enableAutoPerformanceTracking) {
+      result.success(null)
+      return
+    }
+    if (framesTracker == null) {
+      framesTracker = ActivityFramesTracker(LoadClass())
+    }
```

Comment:
We could do that after reading `enableAutoPerformanceTracking` from options, so we know its already created if `enableAutoPerformanceTracking` is enabled.